### DISCUSSION TITLE
JavaSim is interested in joining Commonhaus

### DISCUSSION THREAD

**Discussion by nmcl** [commonhaus/foundation#151] [Joining Commonhaus]
### Project information

- **Project name**: JavaSim
- **Project website**: https://github.com/nmcl/JavaSim
- **Code repository**: https://github.com/nmcl/JavaSim
- **License**: LGPL

### Describe your project

I think JavaSim would be a good project for Commonhaus. It used to reside on Codehaus until that closed down. There's also an older project, C++SIM (https://github.com/nmcl/C--SIM) from which it was derived, which could be useful for historical reasons; this is used in a number of government establishments globally. Back in the Codehaus days both JavaSim and C++SIM had contributors but since moving to a personal repo they dropped off. I'd be more than happy to help drive more contributions and users.

**Comment by ebullient** [comment]
I do have one clarifying question for you, which we did chat about separately. Will you create a GH Organization for this (to include C++SIM as an archived project alongside)?

Please use this discussion (not necessarily this sub-thread) to ask any other questions you may have.

- [ ] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [ ] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [ ] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by nmcl** [comment]
One organisation and C++SIM would be archived within. OK?
